### DISCUSSION TITLE
How to integrated offline map in flutter?

### DISCUSSION THREAD

**Discussion by HackerKing93** [community/community#61452] [Programming Help]
### Select Topic Area

General

### Body

 I am stuck on implementing offline map in flutter.

I want to integrated complete offline map in flutter. 
features:-
1. set destination
2. Able to tracking source to destination in real time 
3. also show the time to reach the destination 

anyone plz helps me to create these I give a source code please implemented these features in my project.

**Comment by stevepopovich** [comment]
Hi @HackerKing93 this forum is for feedback about the native GitHub Mobile app. Do you have a question about that?
